R/track-repos.R

```diff
@@ -0,0 +1,44 @@
+#' Clone all ropensci packages & create `"last_commits.csv"`
+#'
+#' @param pkgs_file `"packages.json"` file for all rOpenSci packages.
+#'
+#' @export
+#'
+track_repos <- function(pkgs_file = "https://raw.githubusercontent.com/ropensci/roregistry/gh-pages/packages.json") {
```

Comment:
Maybe not the best function name. It clones repos _and_ it gets their latest commit at the same time.